client/src/features/calendar/Calendar.js

```diff
@@ -39,17 +39,30 @@ const Calendar = ({ date }) => {
     const fetch = async () => {
       // Database data from server
       const response = await DataService.getAll();
+      // if you want to test the loading behavior
+      // await new Promise(resolve => setTimeout(resolve, 20_000));
```

Comment:
This leaves a little inline test that someone else could use if they wanted to easily test the loading effect on their own. Simply sets the site to wait for 20 seconds arbitrarily.